packages/pkh-tezos/src/authmethod.ts

```diff
@@ -0,0 +1,99 @@
+import { AccountId } from 'caip'
+import { randomString } from '@stablelib/random'
+import { Cacao, SiwTezosMessage, AuthMethod, AuthMethodOpts } from '@didtools/cacao'
+
+export const TEZOS_MAINNET_CHAIN_REF = 'NetXdQprcVkpaWU' // Tezos mainnet beta
+export const TEZOS_DEVNET_CHAIN_REF = 'NetXm8tYqnMWky1' // Tezos devnet beta
+export const VERSION = '1'
+export const CHAIN_NAMESPACE = 'tezos'
+
+export const chainIdMap = {
+  mainnet: TEZOS_MAINNET_CHAIN_REF,
+  devnet: TEZOS_DEVNET_CHAIN_REF,
+}
+
+type TezosNetwork = 'mainnet' | 'devnet'
+
+export namespace TezosWebAuth {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  export async function getAuthMethod(
+    tzProvider: any,
+    account: AccountId,
+    publicKey: string
```

Comment:
earlier comment mentioned pk lookup and network request, the approach now is the dev passes the pubkey here right? and they can look it up how they want 

this approach looks good

Reply:
Yes. Now we have added the getPublicKey helper, directly using tzProvider to get public key